proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/ServerLoginSuccessPacket.java

```diff
@@ -118,6 +121,9 @@ public void encode(ByteBuf buf, ProtocolUtils.Direction direction, ProtocolVersi
         ProtocolUtils.writeProperties(buf, properties);
       }
     }
+    if (version.noLessThan(ProtocolVersion.MINECRAFT_1_20_5)) {
+      buf.writeBoolean(false);
```

Comment:
Might want to add a note that this probably will get removed in 1.21
> Changed the approach to game client disconnecting on invalid data in packets sent from a server
> To ease the transition period, modded servers can opt out by setting the appropriate field in the handshake/game_profile packet. This option will be removed in the next release

Reply:
I don't think a comment about it is so necessary, but I added a system variable to disable that check in the client, by default the check will be enabled